modules/promotion/src/Plugin/Commerce/PromotionCondition/OrderAmountComparison.php

```diff
@@ -0,0 +1,82 @@
+<?php
+
+namespace Drupal\commerce_promotion\Plugin\Commerce\PromotionCondition;
+
+use Drupal\Core\Form\FormStateInterface;
+
+/**
+ * Provides a 'Order: Percentage off' condition.
+ *
+ * @PromotionCondition(
+ *   id = "commerce_promotion_order_amount_comparison",
+ *   label = @Translation("Order Amount Comparison"),
+ *   target_entity_type = "commerce_order",
+ * )
+ */
+class OrderAmountComparison extends PromotionConditionBase {
```

Comment:
Orders don't have an amount, they have a total price. So the class should be OrderTotalPrice, no?